### DISCUSSION TITLE
How to combine app/router.options.ts and routes by pages/

### DISCUSSION THREAD

**Discussion by darklow** [nuxt/nuxt#16279] [Questions]
I'm using `app/router.options.ts` to define some specific group of routes by regexp, but if nothing matches I'd like router to fallback to pages/ structure. Is something like this possible? 

Right now as soon as I define `app/router.options.ts` then if nothing matches I see 404 error and there is no fallback to pages/ structured routes.

Thanks.

**Comment by FileSoft-us** [comment]
`import type { RouterOptions } from '@nuxt/schema';

export default <RouterOptions>{
  routes: (_routes) => {
     //do everythings... 
     you can map the route in a function then return it
  }
}`
